simple/deploy.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
+set -e
+
+ECHO=`which echo`
+KUBECTL=`which kubectl`
+
+# methods
+function echoBold () {
+    ${ECHO} -e $'\e[1m'"${1}"$'\e[0m'
+}
+
+function usage () {
+    echoBold "This script automates the installation of WSO2 Identity Server with Analytics Kubernetes resources\n"
+    echoBold "Allowed arguments:\n"
+    echoBold "-h | --help"
+    echoBold "--wu | --wso2-username\t\tYour WSO2 username"
+    echoBold "--wp | --wso2-password\t\tYour WSO2 password"
+    echoBold "--cap | --cluster-admin-password\tKubernetes cluster admin password\n\n"
+}
+
+WSO2_SUBSCRIPTION_USERNAME=''
+WSO2_SUBSCRIPTION_PASSWORD=''
+ADMIN_PASSWORD=''
+
+# capture named arguments
+while [ "$1" != "" ]; do
+    PARAM=`echo $1 | awk -F= '{print $1}'`
+    VALUE=`echo $1 | awk -F= '{print $2}'`
+
+    case ${PARAM} in
+        -h | --help)
+            usage
+            exit 1
+            ;;
+        --wu | --wso2-username)
+            WSO2_SUBSCRIPTION_USERNAME=${VALUE}
+            ;;
+        --wp | --wso2-password)
+            WSO2_SUBSCRIPTION_PASSWORD=${VALUE}
+            ;;
+        --cap | --cluster-admin-password)
+            ADMIN_PASSWORD=${VALUE}
+            ;;
+        *)
+            echoBold "ERROR: unknown parameter \"${PARAM}\""
+            usage
+            exit 1
+            ;;
+    esac
+    shift
+done
+
+# create a new Kubernetes Namespace
+${KUBECTL} create namespace wso2
+
+# create a new service account in 'wso2' Kubernetes Namespace
+${KUBECTL} create serviceaccount wso2svc-account -n wso2
```

Comment:
Can't we define the Service account as a YAML config?

Reply:
We can. 
But since we can't create the secrete in yaml we need to include it in a bash anyway. If we create the service-account in yaml we may need to run 3 file. 
1. yaml file for namespace and service-account
2. "create secret" bash command and
3. yaml file for configurations and deployments.

Reply:
I meant define service account in the YAML file referred in point 3. Why can't we create the Secret in the YAML file?.